CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.70.4 - 2022-08-23
+
+What's new:
+- Improve commit log performance for single match by adding limit flag (#3970)
```

Comment:
This adds a flag, so requires a minor version bump.

(Still **NOT** a semver violation, because we're pre-1.0.0, and all the places behave as though shifted by one place...).